e2e_tests/tests/cluster/test_master_restart.py

```diff
@@ -15,31 +14,13 @@
 from tests import config as conf
 from tests import detproc
 from tests import experiment as exp
-from tests.cluster import (
-    abstract_cluster,
-    managed_cluster,
-    managed_cluster_k8s,
-    managed_slurm_cluster,
-    utils,
-)
+from tests.cluster import abstract_cluster, managed_cluster, managed_cluster_k8s, utils
```

Comment:
my interpretation of the google python style guide would make all the local imports are all wrong, haha. so i wonder what @rb-determined-ai has to say?

Reply:
im not even sure if we follow google python, tbh.

Reply:
[pep8](https://peps.python.org/pep-0008/#imports) doesn't 100% clarify either unfortunately. I did not change this by hand, this was a change from `black` and/or `isort` when I removed `ManagedSlurmCluster`. 

My personal take is style guides don't matter unless enforced by compiler or formatter. We can tweak it but it will probably revert back to this form the next time the formatter runs.

Reply:
As a matter of fact, we do have a [style guide](https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md).  This is more about how to write code than how to import it afterwards.  When importing, we follow the [google style guide for imports](https://google.github.io/styleguide/pyguide.html#22-imports), which we actually do lint for.  We don't follow the whole google style guide.

> My personal take is style guides don't matter unless enforced by compiler or formatter

I suppose it depends on what you call "style".  For things like indentation, I'd probably agree with @djanicekpach.  For things like "how do you design an API", those are more a matter of taste and you have to teach people how to do it, then it becomes a core part of their job.  And how you import is kinda related to how you design an API because the way you structure your python code directly affects how consumers of that code import it, which is what our style guide is all about.

> I goes from one import per line even with repeated module

I _think_ this is related to python syntax limitations around renaming `experiment as exp`.  There are some "canonical" names in our `e2e_tests/` directory that are pretty dumb, but just haven't been fixed.  `experiment as exp` is the worst offender of all.